packages/react-client/src/ReactFlightClient.js

```diff
@@ -1721,9 +1731,13 @@ function createFakeFunction<T>(
 }
 
 const frameRegExp =
-  /^ {3} at (?:(.+) \(([^\)]+):(\d+):(\d+)\)|([^\)]+):(\d+):(\d+))$/;
+  /^ {3} at (?:(.+) \(([^\)]+):(\d+):(\d+)\)|(?:async )?([^\)]+):(\d+):(\d+))$/;
```

Comment:
Can we add one or two examples as comments here? Doesn't need to be super exhaustive. It's just a very intimidating regex 😬

Reply:
Added a comment but I'm still ambivalent about how much we should cover here. This doesn't really cover all possible.